### PR TITLE
Add samtools cramsize

### PR DESCRIPTION
**Pull request by SPPearce**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by SPPearce** [comment]
Splitting off the samtools cram-size from https://github.com/nf-core/modules/pull/3555

File: modules/nf-core/samtools/cramsize/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    tag "samtools"
+    tag "samtools/cramsize"
+
+    test("human - cram") {
+
+        when {
+            process {
+                """
+                input[0] = [[ id:'test', single_end:false ], // meta map
+                            file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_cram'], checkIfExists: true)
```
Comment (CarsonJM) [inline modules/nf-core/samtools/cramsize/tests/main.nf.test:18]:
```suggestion
                            file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/cram/test.paired_end.sorted.cram', checkIfExists: true)
```

File: modules/nf-core/samtools/cramsize/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    test("human - cram - stub") {
+
+        options "-stub"
+
+        when {
+            process {
+                """
+                input[0] = [[ id:'test', single_end:false ], // meta map
+                            file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_cram'], checkIfExists: true)
```
Comment (CarsonJM) [inline modules/nf-core/samtools/cramsize/tests/main.nf.test:41]:
```suggestion
                            file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/cram/test.paired_end.sorted.cram', checkIfExists: true)
```

**Review by CarsonJM — APPROVED** [review]
LGTM! Just a teeny change regarding how the test data path is specified.

**Pull request merged**